src/test/java/net/mcreator/integration/TestWorkspaceDataProvider.java

```diff
@@ -473,8 +473,8 @@ private static GeneratableElement getExampleFor(ModElement modElement, boolean u
 			fluid.generateBucket = !_true;
 			fluid.bucketName = modElement.getName() + " Bucket";
 			fluid.textureBucket = emptyLists ? "" : "itest";
-			fluid.creativeTab = new TabEntry(modElement.getWorkspace(),
-					getRandomDataListEntry(random, ElementUtil.loadAllTabs(modElement.getWorkspace())));
+			fluid.creativeTabs = List.of(new TabEntry(modElement.getWorkspace(),
+					getRandomDataListEntry(random, ElementUtil.loadAllTabs(modElement.getWorkspace()))));

```

Comment:
Maybe you should test with multiple creative tabs in case there's a change someday we don't know/are not aware/didn't think of.

Reply:
Thanks for the catch, should I replace it with list of all tabs (vanilla and custom)?

Reply:
Testing with all creative tabs could help catch more problems I think, but at least one vanilla and one custom should be tested at the same time.

Reply:
I noticed one problem. As `ModElementType.TAB` is registered after most item-type mod elements, custom tabs will only be visible when configuring tool elements. To include them in the tabs list from the very beginning, I'd need to move tab MET initializer to the top of the registry, but these are sorted in alphabetical order 🤷‍♂️

Reply:
To oversome this problem, we have TestWorkspaceDataProvider.fillWorkspaceWithTestData